bin/prepare-kubectl

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
+  . k8s-read-config "$@"
+
 mkdir -p ~/.kube
 if [ -n "$KUBECONFIG_DATA" ]; then
-  echo $KUBECONFIG_DATA | base64 --decode > ~/.kube/config
+  echo "${KUBECONFIG_DATA}" | base64 --decode > ~/.kube/config
+else
+  cloud-login
+  gcloud container clusters get-credentials "${CLUSTER_NAME}" --zone "${GCP_ZONE}" --project "${GCP_PROJECT}"
```

Comment:
This may need an another `if` to check that `gcloud` should be used. It looks like `cloud-login` will silently exit if there are no vars for logging in. This may lead to `gcloud` being called unexpectedly for users who do not use GCP.